android/app/src/main/assets/custom/index.ts

```diff
@@ -0,0 +1 @@
+export * from './';
```

Comment:
Add source folder within fonts and check if this folder will be removed

Reply:
It is Fixed and it is working the same now.